desc/compute/_metric.py

```diff
@@ -2088,3 +2088,350 @@ def _cvdrift0(params, transforms, profiles, data, **kwargs):
         * dot(data["b"], cross(data["grad(|B|)"], data["e^rho"]))
     )
     return data
+
+
+################################################################################
+##########-----------------METRIC ELEMENTS PEST----------------------###########
+################################################################################
+
+
+@register_compute_fun(
+    name="g_rr|PEST",
+    label="g_{\\rho\\rho}|PEST",
+    units="m^{2}",
+    units_long="square meters",
+    description="Radial/Radial element of covariant metric tensor",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["e_rho|v,p"],
+)
+def _g_sub_rr_PEST(params, transforms, profiles, data, **kwargs):
+    data["g_rr|PEST"] = dot(data["e_rho|v,p"], data["e_rho|v,p"])
+    return data
+
+
+@register_compute_fun(
+    name="g_rt|PEST",
+    label="g_{\\rho\\theta}|PEST",
+    units="m^{2}",
+    units_long="square meters",
+    description="Radial/Radial element of covariant metric tensor",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["e_rho|v,p", "e_theta_PEST"],
+    aliases=["g_tr|PEST"],
+)
+def _g_sub_rt_PEST(params, transforms, profiles, data, **kwargs):
+    data["g_rt|PEST"] = dot(data["e_rho|v,p"], data["e_theta_PEST"])
+    return data
+
+
+@register_compute_fun(
+    name="g_rz|PEST",
+    label="g_{\\rho\\phi}|PEST",
+    units="m^{2}",
+    units_long="square meters",
+    description="Radial/Radial element of covariant metric tensor",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["e_rho|v,p", "e_phi|r,v"],
+    aliases=["g_zr|PEST"],
+)
+def _g_sub_rz_PEST(params, transforms, profiles, data, **kwargs):
+    data["g_rz|PEST"] = dot(data["e_rho|v,p"], data["e_phi|r,v"])
+    return data
+
+
+@register_compute_fun(
+    name="g_tt|PEST",
+    label="g_{\\theta_PEST \\theta_PEST}|PEST",
+    units="m^{2}",
+    units_long="square meters",
+    description="Radial/Radial element of covariant metric tensor",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["e_theta_PEST"],
+)
+def _g_sub_tt_PEST(params, transforms, profiles, data, **kwargs):
+    data["g_tt|PEST"] = dot(data["e_theta_PEST"], data["e_theta_PEST"])
+    return data
+
+
+@register_compute_fun(
+    name="g_tz|PEST",
```

Comment:
add note/TODO that this is assuming zeta=phi

Reply:
Ok.